### PR TITLE
chore: added common control pt-navigator

### PR DESCRIPTION
**Pull request by LucyGit**

### CODE REVIEW

File: packages/pt-navigator/src/YOUIBreadcrumbs.vue
Comment (idoprz) [inline packages/pt-navigator/src/YOUIBreadcrumbs.vue:1]: Please change the name of this. It should be generic about tabs vs stepper and not directly related to specific consumer like yeomanui.

File: packages/pt-navigator/src/PTNavigator.vue
Comment (idoprz) [inline packages/pt-navigator/src/PTNavigator.vue:1]: What is the meaning of PT? why is it called "PT"?

**Review by idoprz — COMMENTED** [review]
Hi,
1. In general I would like to see a full simple example on how to use in the examples folder
2. Regarding the names look at my comments, it should be generic (tabs/steps) not specific to yeoman.
3. I don't really understand what is PT.

File: packages/pt-navigator/src/PTNavigator.vue
Comment (LucyGit) [reply packages/pt-navigator/src/PTNavigator.vue:1]: PT means "Productivity Tools"

Comment (idoprz) [reply packages/pt-navigator/src/PTNavigator.vue:1]: I don't think it's clear. Also not sure why a user control related to productivity tools. If you can you should change it to something more meaning full like what is this control.. For example StepControl or something like this.

Comment (LucyGit) [reply packages/pt-navigator/src/PTNavigator.vue:1]: As I used in the help, I named it as "Navigator", but I think it is too common, so I added the prefix. What do you think about "Navigator"

Comment (LucyGit) [reply packages/pt-navigator/src/PTNavigator.vue:1]: The package name is "@sap-devx/pt-navigator" currently. 
I think of the package name is ""@sap-devx/controls-navigator", and the control name is "Navigator". What do you think about it

Comment (LucyGit) [reply packages/pt-navigator/src/PTNavigator.vue:1]: Hi Ido, what's your opinion about the control name?

**Pull request closed without merging**